openfl/utilities/dataloading.py

```diff
@@ -35,7 +35,7 @@ def get_dataloader(
             Defaults to `None`.
         collaborator_index (int ?):
             which collaborator should be used for initializing dataloader
-            among collaborators specified in plan/data.yaml.
```

Comment:
I'm not sure I understand the need for these changes...

Reply:
Reverted